lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1003,106 +982,99 @@ export class UserAgentApplication {
   @resolveTokenOnlyIfOutOfIframe
   acquireTokenSilent(scopes: Array<string>, authority?: string, user?: User, extraQueryParameters?: string): Promise<string> {
     return new Promise<string>((resolve, reject) => {
-      // Validate scopes
-      // TODO: Replace with new validation pattern
-      const isValidScope = this.validateInputScope(scopes);
-      if (isValidScope && !Utils.isEmpty(isValidScope)) {
+      // Validate and filter scopes
```

Comment:
The changes here are simply that the code below was extracted out of the else block of the condition. No other code logic changes were made.